packages/browser-agent/README.md

```diff
@@ -1,27 +1,148 @@
 
-It provides APIs to initialize the agent:
+# New Relic Browser Agent
 
+The New Relic Browser Agent instruments your website and provides observability into the performance and behavior of your application. This NPM Library is an ***in-progress*** implementation of the New Relic Browser Agent, accessible through `NPM`. Please see the [Differences](#differences) and [Features](#features) sections to compare how this library differs from the other offerings of the New Relic Browser Agent.
+
+## Installation
+
+### Using Package Managers
+
+Using [npm](https://npmjs.org):
+
+```bash
+npm install @newrelic/browser-agent
+```
+
+or [yarn](https://yarnpkg.com/)
+
+```bash
+yarn add @newrelic/browser-agent
+```
+
+### Directly in HTML/JS
+
+Using [unpkg](https://unpkg.com/)
+
+```html
+<script src="https://unpkg.com/@newrelic/browser-agent/bundled"></script>
 ```
-import nr from 'nr-browser-core'
 
-nr.setConfiguration({
-  jserrors: {
-    harvestTimeSeconds: 30
-  }
+## Usage Examples
+
+### Basic Setup
+
+```javascript
+<< App.js >>
+// initialize the agent as close to 
+// the top level of the application as possible
+import NR from '@newrelic/browser-agent'
+const options = {
+    // See 'Configuring your application'
+}
+NR.start(options).then(() => {
+    console.log("Browser Agent Initialized!")
 })
+```
+
+### Instrumenting a Micro Front End
 
-nr.init({
-  beaconUrl: 'localhost:8181',
-  licenseKey: 'abc123',
-  appId: 2
+The New Relic Browser Agent can maintain separate configuration scopes for each generated bundle.  Multiple bundles, like those used in certain micro front end patterns, can each report their own scoped data to separate New Relic applications.
+
+```javascript
+// <<< MICRO FRONT END APP 1 >>>
+// ---- App.js ----
+import NR from '@newrelic/browser-agent'
+const options = {
+    // see 'Configuring your application'
+}
+NR.features.JSERRORS.enabled = true
+NR.start(options).then(() => {
+    console.log("Browser Agent Initialized!")
 })
+
+// ---- MyComponent.js ----
+class MyComponent() {
+    try {
+        ...
+    } catch(err) {
+        NR.noticeError(err)
+        // reports to applicationID 1
+    }
+}
 ```
 
-### Future work / ideas
+## Configuring your application
+
+The NR interface's `start` method accepts an `options` object to configure the agent:
+
+```js
+const options = {
+  licenseKey: String
+  applicationID: String
+  beacon: String
+}
+NR.start(options)
+```
+
+### Get application ID, license key, beacon
+
+You can find `licenseKey`, `applicationID` and `beacon` values in the New Relic UI's Browser Application **Settings** page ([one.newrelic.com](https://one.newrelic.com) > Browser > (select an app) > Settings > Application settings.)
+
+![configuration](https://user-images.githubusercontent.com/4779220/169617807-110f3938-8af9-4aa8-b651-7712589b0792.jpg)
+
```

Comment:
Up For discussion -- We could possibly use another section here just to briefly outline some -- or -- all of the properties that could go into the `options` object ?

| Property | Type |
| --------- | ---------|
| applicationID | string |
| licenseKey | string |
| beacon | string 
| errorBeacon | string |
| sa | number |
| queueTime | number | 
| applicationTime | number |
| tGuid | string |
| user | string |
| account | string |
| product | string |
| extra | string |
| userAttributes | string | 
| atts | string |
| transactionName | string |
| tNamePlain | string |
| privacy | { cookies_enabled?: boolean } |
| ajax | {deny_list?: string[]} |
| distributed_tracing | { enabled?: boolean,  exclude_newrelic_header?: boolean, cors_use_newrelic_header?: boolean, cors_use_tracecontext_headers?: boolean, allowed_origins?: string[] } |
| page_view_timing | { enabled?: boolean } |
| obfuscate | {regex?: RegExp, replacement?: string}[] |

Reply:
That's definitely worth coming back to, especially for the feature specific configs as they're implemented. For now, privacy, ajax, distributed_tracing, page_view_timing have no impact on the errors feature, obfuscate is a work in progress, errorBeacon is treated identically to beacon, and the rest are set by APM and aren't relevant for this package.